.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: Build Swift Package
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
So this is both a PR check and merge condition?

Reply:
Yeah, I figured might as well verify on both before and after the merge it's still all good (only takes a minute to build).

Reply:
Ok works for me

Reply:
It's probably overkill by a lot at the moment but may be more useful down the road.